### DISCUSSION TITLE
Welcome to MERN-E-Commerce Discussions!

### DISCUSSION THREAD

**Discussion by shubham1710** [shubham1710/MERN-E-Commerce#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rudrachaubey** [comment]
bro there is a error showing proptypes 
Warning: Failed prop type: The prop `user` is marked as required in `Home`, but its value is `null`
[HMR] Waiting for update signal from WDS...
